### PR TITLE
clean the code - noqa and pragma

### PR DESCRIPTION
**Pull request by Mitchina**

### CODE REVIEW

File: {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py [hunk 1]
```diff
@@ -90,10 +90,10 @@ class ProjectDefault(Configuration):
 
     # Password validation
     # https://docs.djangoproject.com/en/stable/ref/settings/#auth-password-validators
-    # fmt: off
+
     AUTH_PASSWORD_VALIDATORS = [
         {
-            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",  # noqa
+            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator", # noqa: E501
```
Comment (pauloxnet) [inline {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py:96]:
```suggestion
            "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",  # noqa: E501
```
Leave two spaces before comment

**Review by pauloxnet — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py (hunk 1 above)
Comment (Mitchina) [reply {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py:96]: okk

**Comment by trottomv** [comment]
These changes are too related to upgrade versions and tools of linter checks. 
It should either be expanded or introduced in conjunction with check linter tool and dependency updates

**Review by pauloxnet — APPROVED** [review]

File: {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py [hunk 2]
```diff
@@ -389,7 +386,7 @@ def DATABASES(self):
     # https://sentry.io/for/django/
 
     try:
-        import sentry_sdk  # noqa
+        import sentry_sdk
```
Comment (filippo-20tab) [inline {{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py:389]: Sentry is often present but not always. Therefore, I suggest what follows
```suggestion
        import sentry_sdk  # noqa: F401
```

**Review by pauloxnet — APPROVED** [review]

**Pull request merged**